index.js

```diff
@@ -268,32 +293,28 @@ client.on('ready', async () => {
   }
 });
 
-
 const loadCredentials = () => {
   // Load client secrets from a local file.
   try {
-    return content = JSON.parse(fs.readFileSync('credentials.json'))
+    const content = JSON.parse(fs.readFileSync('credentials.json'));
+    return content;
```

Comment:
Instead of splitting into two lines we should just `return JSON.parse(fs.readFileSync('credentials.json'));` We don't need to assign that value to any variable.

Reply:
Done.